CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@
 Changelog
 =========
 
+v4.8.1 (2020-10-20)
```

Comment:
I generally don't add release version info until I'm ready to release. Can just leave this section heading out.